### PR TITLE
Priyasbranch

### PR DESCRIPTION
**Pull request by panamadan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -121,12 +121,10 @@ <h3 id="filterHeader">Filter</h3>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
     <!-- Compiled and minified JavaScript -->
     <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
-    <script src="j.js"></script>
```
Comment (pmahalan) [inline index.html:181]: You can delete this line, I renamed the JS file to mmlapis.js.
(We don't need the j.js file anymore.)

File: index.html [hunk 2]
```diff
@@ -121,12 +121,10 @@ <h3 id="filterHeader">Filter</h3>
...
-
                 
     <!--JavaScript at end of body for optimized loading-->
-    <script type="text/javascript" src="js/materialize.min.js"></script>
-    <script src="j.js"></script>
```
Comment (pmahalan) [inline index.html:186]: We don't need line 129 either, or line 124

File: index.html [hunk 3]
```diff
@@ -121,12 +121,10 @@ <h3 id="filterHeader">Filter</h3>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
     <!-- Compiled and minified JavaScript -->
     <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
-    <script src="j.js"></script>
-
                 
     <!--JavaScript at end of body for optimized loading-->
-    <script type="text/javascript" src="js/materialize.min.js"></script>
-    <script src="j.js"></script> 
+    <!-- <script type="text/javascript" src="js/materialize.min.js"></script>  -->
```
Comment (pmahalan) [inline index.html:183]: I don't think we need line 126 because we already have the exact same thing on line 128

**Review by pmahalan — COMMENTED** [review]
We have a couple repeats in the .html document; I changed the name of the .js file, so we just have a couple lines to revise between 124 and 129 (roughly) in the HTML doc.

**Review by pmahalan — CHANGES_REQUESTED** [review]
I thought I already added the review but GH is saying it still needs one

**Review by panamadan — COMMENTED** [review]
everything has been resolved

**Review by panamadan — COMMENTED** [review]
changed

**Pull request closed without merging**